### ISSUE TITLE
No label for screen readers.

### ISSUE THREAD

**Issue by kalaspuffar** [PolymerElements/iron-autogrow-textarea#98]
### Description
iron-autogrow-textarea has no label for screen readers to read so when entering the field NVDA is quiet.

### Expected outcome

NVDA should read the label of the element.

### Actual outcome

NVDA is silent.

### Steps to reproduce

1. Add one or more input-textarea on the page with a label.
2. Start NVDA
3. Visit the page and tab through the inputs.

### Browsers Affected
- [x] Chrome
- [ ] Firefox
- [ ] Safari 9
- [ ] Safari 8
- [ ] Safari 7
- [ ] Edge
- [ ] IE 11
- [ ] IE 10

**Comment by kalaspuffar** [comment]
I know that this might not be the correct solution all the way because the old implementation also added information about counter and errors to the aria-describedby. But these values could be added as well if we want to continue with this direction. I just wanted to open the issue with some code the start the discussion about how to solve this issue.

We work daily to create accessible media and want to contribute to a more accessible web, so I hope this contribution is furthering that goal.

**Comment by danbeam** [comment]
this is also affecting Chrome's settings UI

https://bugs.chromium.org/p/chromium/issues/detail?id=700611

@notwaldorf
